site/docs/v1/tech/admin-guide/upgrade.adoc

```diff
@@ -4,44 +4,300 @@ title: Upgrade FusionAuth
+
+FusionAuth can perform database migrations automatically and silently using a process called link:/docs/v1/tech/guides/silent-mode[Silent Mode]. 
+
+If you want to run the migrations interactively, you can run any needed scripts out of band (meaning directly on the database yourself). The general process would be to: 
+
+- Remove all server nodes from a load balancer/proxy.
+- Run the database migration scripts.
+- Add all server nodes back to the load balancer/proxy.
 
 [NOTE]
 ====
-*Readme*
+You should always back up your database before using Maintenance Mode or performing any database migrations by any method.
+====
 
-To upgrade to the latest version of FusionAuth using the FastPath installation go link:/docs/v1/tech/installation-guide/fast-path[here].
+The timing of the database upgrade is also important to note. When an upgraded node is started and connects to the database, it will lock the database immediately if configured for Silent Mode and the database requires upgrading. This means that the database will be unavailable to other nodes in the cluster until the upgrade is complete.
 
-To upgrade to the latest version of FusionAuth using Homebrew go link:/docs/v1/tech/installation-guide/homebrew[here].
+=== Connected Applications and Client Libraries
 
-To upgrade to the latest version of FusionAuth using Docker go link:/docs/v1/tech/installation-guide/docker[here].
+Consider that a FusionAuth server upgrade might mean that you need to upgrade your client libraries too. Check if this is needed before upgrading, as it will entail a more complex upgrade process and may require more downtime.
 
-If you want to use manual mode, download the latest version of FusionAuth and follow the instructions below. See https://fusionauth.io/download[Downloads].
+In general, client libraries are forward-compatible, but any issues will be noted in the release notes. (Read the release notes carefully.)
+
+=== Skipping Versions 
+
+Sometimes, you will want to upgrade to a FusionAuth version that is a few versions newer than your current version. This is called a "version jump". Version jumps include minor versions. FusionAuth migration can run all the necessary database migrations to get you to the latest version. 
+
+In some cases, version jumps are safe to do. To determine if you can perform a version jump, you should review the link:/docs/v1/tech/release-notes[release notes] carefully. If the releases focus on bug fixes and minor improvements, you can probably perform a version jump safely. Additionally, if the only changes are to features that you are not using, you can probably perform a version jump.
+
```

Comment:
```suggestion
The alternative to a version jump is to migrate and test at each version of FusionAuth. For example, if you were moving from 1.43.0 to 1.45.0, you'd upgrade to 1.43.1 and test, then 1.43.2 and test, then 1.43.3 and test, then 1.44.0 and test, then 1.45.0 and test. This is a good option because if there are any issues, you'll be able to know precisely which version of FusionAuth is problematic. It's a great option if you have automated tests that can exercise FusionAuth integrations.


```